gemd/units/impl.py

```diff
@@ -98,7 +160,7 @@ def _format_clean(unit, registry, **options):
 
 
 @functools.lru_cache(maxsize=1024)
-def parse_units(units: Union[str, Unit, None]) -> Union[str, Unit, None]:
+def _parse_units(units: str) -> Unit:
```

Comment:
Method with clean signature for internal units conversion.

Reply:
Hopefully, this routine goes away and we can just use `UnitRegistry.parse_units` once we stop supporting python 3.7.